README.md

```diff
@@ -16,6 +16,14 @@ See [Choosing Secure Passwords](https://www.schneier.com/blog/archives/2014/03/c
 This feature enables the HTTP header `X-XSS-Protection: 1; mode=block` to be sent on all requests which some web browsers intend as a way to automatically block suspected cross-site scripting attacks.
 Several web browsers (e.g., Firefox, Edge, and Chrome) do not support this header.
 
+## Remove HTTP Headers for Unauthorized Users
+
+This feature adds a configurable list of HTTP header names (case-insensitive) that can be automatically removed from HTTP responses sent to users lacking the *Overall/Read* permission.
+This is useful for hiding the Jenkins and other software versions present in HTTP headers like `X-Jenkins` and `Server`.
```

Comment:
I don't think this can be used to filter out "Server". At least it didn't work when I tried it.